plutus-core/untyped-plutus-core/test/Evaluation/Builtins/Definition.hs

```diff
@@ -458,107 +464,161 @@ test_SerialiseDataImpossible =
         typecheckAnd def evalRestricting defaultBuiltinCostModelForTesting dataLoop @?=
             Right EvaluationFailure
 
+stripParensIfAny :: String -> String
+stripParensIfAny ('(' : str) = init str
```

Comment:
So what this does is removing the first two characters if the first character is `(`. Worth a comment on why it does that.

Reply:
Made it

```haskell
-- | If the first char is an opening paren and the last chat is a closing paren, then remove them.
-- This is useful for rendering a term-as-a-test-name in CLI, since currently we wrap readably
-- pretty-printed terms in parens (which is to be fixed).
stripParensIfAny :: String -> String
stripParensIfAny str@('(' : str1) | last str == ')' = init str1
stripParensIfAny str                                = str
```